File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/localevent/ArenaEventHandler.java
package com.gmail.borlandlp.minigamesdtools.arena.localevent;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@Retention(RetentionPolicy.RUNTIME)
public @interface ArenaEventHandler {
    ArenaEventPriority priority() default ArenaEventPriority.LOWEST;
    boolean ignoreCancelled() default false;
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/localevent/ArenaEventHandler.kt
package com.gmail.borlandlp.minigamesdtools.arena.localevent

@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
annotation class ArenaEventHandler(
    val priority: ArenaEventPriority = ArenaEventPriority.LOWEST,
    val ignoreCancelled: Boolean = false
)